app/assets/v2/js/grants/index.js

```diff
@@ -102,7 +118,7 @@ Vue.component('grant-sidebar', {
           collections_query.keyword = params.keyword;
         }
 
-        document.location.href = `/grants/collections?${$.param(collections_query)}`;
+        // document.location.href = `/grants/collections?${$.param(collections_query)}`;
```

Comment:
This is because this PR is against master instead of stable, right? Should we put this against master?

Reply:
yes ! let's switch this to master

Reply:
Do we need the return on line 113?